tests/phpunit/tests/user/getUserCount.php

```diff
@@ -0,0 +1,164 @@
+<?php
+
+/**
+ * @group user
+ */
+class Tests_User_GetUserCount extends WP_UnitTestCase {
+	/**
+	 * @ticket 40386
+	 * @group multisite
+	 * @group ms-required
+	 */
+	public function test_wp_update_network_counts_on_different_network() {
+		$different_network_id = self::factory()->network->create(
+			array(
+				'domain' => 'wordpress.org',
+				'path'   => '/',
+			)
+		);
+
+		delete_network_option( $different_network_id, 'user_count' );
+
+		wp_update_network_counts( $different_network_id );
+
+		$user_count = (int) get_user_count( $different_network_id );
```

Comment:
```suggestion
		$user_count = get_user_count( $different_network_id );
```
`get_user_count()` already returns `int`.